gcp_variant_transforms/vcf_to_bq_common_test.py

```diff
@@ -87,3 +87,21 @@ def test_default_optimize_for_large_inputs(self):
     with mock.patch.object(FileSystems, 'match', return_value=[match]):
       self.assertEqual(vcf_to_bq_common.get_pipeline_mode(args.input_pattern),
                        PipelineModes.MEDIUM)
+
+  def test_fail_on_unrecognized_flags_1(self):
```

Comment:
nit: instead of '1' and '2', consider renaming to `test_fail_on_unrecognized_flags` and `test_fail_on_unrecognized_flags_no_failure`.

Reply:
Done. Thanks.